### PR TITLE
Verify Interface Status

### PR DESCRIPTION
**Pull request by axkar**
Add a test case to verify interface status (enable/disable)

## Checklist

Tick *relevant* boxes, this PR is-a or has-a:

- [ ] Bugfix
  - [ ] Regression tests
  - [ ] ChangeLog updates (for next release)
- [ ] Feature
  - [ ] YANG model change => revision updated?
  - [ ] Regression tests added?
  - [ ] ChangeLog updates (for next release)
  - [ ] Documentation added?
- [x] Test changes
  - [x] Checked in changed Readme.adoc (make test-spec)
  - [x] Added new test to group Readme.adoc and yaml file
- [x] Code style update (formatting, renaming)
- [x] Refactoring (please detail in commit messages)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

### CODE REVIEW

File: test/case/ietf_interfaces/iface_enable_disable/test.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+"""
+Interface status
+
+Verify that interface status works
```
Comment (troglobit) [inline test/case/ietf_interfaces/iface_enable_disable/test.py:5]: Maybe: Verify interface status properly propagate changes when an interface is disabled and then re-enabled. Both admin-status and oper-status is verified.

**Review by troglobit — CHANGES_REQUESTED** [review]

File: test/case/ietf_interfaces/iface_enable_disable/test.py (hunk 1 above)
Comment (axkar) [reply test/case/ietf_interfaces/iface_enable_disable/test.py:5]: Thnx! I'll take it :)

Comment (troglobit) [reply test/case/ietf_interfaces/iface_enable_disable/test.py:5]: I think it should be propagates, plural.

File: test/case/ietf_interfaces/iface_enable_disable/Readme.adoc [hunk 2]
```diff
@@ -0,0 +1,28 @@
+=== Interface status
+==== Description
+Verify that interface status works
+
```
Comment (troglobit) [inline test/case/ietf_interfaces/iface_enable_disable/Readme.adoc:4]: I guess we should update the generated files after that description changes.

**Review by troglobit — CHANGES_REQUESTED** [review]

File: test/case/ietf_interfaces/iface_enable_disable/Readme.adoc (hunk 2 above)
Comment (axkar) [reply test/case/ietf_interfaces/iface_enable_disable/Readme.adoc:4]: Oh yes. I still forget that :( ....

Comment (axkar) [reply test/case/ietf_interfaces/iface_enable_disable/Readme.adoc:4]: Coming

**Review by troglobit — APPROVED** [review]

**Pull request merged**